lib/cc/cli/init.rb

```diff
@@ -74,27 +74,40 @@ def create_codeclimate_checks
 
       def create_default_engine_configs
         say "Generating default configuration for engines..."
-        available_engine_configs.each do |config_path|
-          file_name = File.basename(config_path)
-          if filesystem.exist?(file_name)
-            say "Skipping generating #{file_name} file (already exists)."
-          else
-            filesystem.write_path(file_name, File.read(config_path))
-            success "Config file #{file_name} successfully generated."
+        available_engine_configs.each do |(engine_name, config_paths)|
```

Comment:
Not strictly in the latest Rubies. I personally prefer it that way as in this case it's destructuring. `available_engine_configs` is an array of arrays so block is technically passed a single argument, an array of two values.